### PR TITLE
Request to merge the home page for the apple clone project to master.

### PR DESCRIPTION
**Pull request by ClaytonSiby**
The apple-clone branch is a replica of the old version of the apple website, it is designed using HTML  and CSS. The page is not responsive, this is based on the requirements for the project from the curriculum.

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required!
Hi @ClaytonSiby, impressive job so far! Nice work with the navigation gradient! :+1: There a few issues for you to fix to complete this project. These include;
### Mandatory:
- [x] Your navigation appears broken on my screen. Look into your use of percentages for sizing and `px`s for positioning. ([Screenshot](https://github.com/trekab/images/blob/master/claytonAppleCloneNav.png?raw=true))
- [x] The footer and bottom images also appear broken. Check out [this screenshot](https://github.com/trekab/images/blob/master/claytonAppleClone.png?raw=true).
- [x] The bottom four elements with images should be links.
- [x] You have repeated styles in your CSS. `.apple-image-1, 2, & 3` have the same repeated rules and `background-size: cover cover;` has an invalid property.

### Optional:
- [x] Make use of [CSS validator](https://jigsaw.w3.org/css-validator/) to help you catch most CSS errors.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach on slack for any inquiries about this review_ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

File: style.css [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+  height: 100%;
+}
+
+.apple-img-4 {
+  background-image: url("images/image4.jpg");
+  background-size: cover cover;
+  background-repeat: no-repeat;
+  width: 25%;
+  height: 100%;
+}
```
Comment (trekab) [inline style.css]: This repetition can be refactored and `cover cover` is not a CSS property.

Comment (ClaytonSiby) [reply style.css]: Hello trekab

Thank you for your review. It is much appreciated and very helpful. I have done all the changes, however, I cannot give my `.apple-image-1, 2, & 3` the same rule as they have different background images. I am open to any suggestions and kind regards.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 80vh;
+}
+
+.navbar {
+  border-bottom-left-radius: 0.5rem;
+  border-top-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+  border-top-right-radius: 0.5rem;
```
Comment (christianotieno) [inline style.css]: Please refactor this line with this suggestion.
```suggestion
border-radius: 0.5rem;
```

Comment (christianotieno) [reply style.css]: Consider these changes optional.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project. :fireworks: Keep it up.
##
## OPTIONAL CHANGES:
- [x] Consider some refactoring in the comments below.
- [x] Remove the scroll in the page, please.
##
- Carry out the changes then later merge this. You don't have to submit another code review for this.  I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**